.github/workflows/zfs-tests-functional.yml

```diff
@@ -78,5 +75,5 @@ jobs:
       if: failure()
       with:
         name: Test logs Ubuntu-${{ matrix.os }}
-        path: /var/tmp/test_results/20*/
+        path: /var/tmp/test_results/*
```

Comment:
IIRC there is a 'current_logs' symlink here, Github's action for artefacts don't work well with symlinks, could you, please, show output archive?

Reply:
Zipping full `/var/tmp/test_results` content seems more robust. Indeed, there's a symlink inside, it doesn't looks like it is causing troubles.

https://www.sendspace.com/file/gipv5l

Reply:
Then let's just compress `/var/tmp/test_results/current`? If it works now with github actions' artifacts.

Reply:
In the case of re-run (`-R`) we have 3 separate directories in `/var/tmp/test_results`:
- original run logs
- re-run logs
- the `current` symlink

Reply:
Oh, you're right, forgot about re-runs, then maybe something like that may be the best one
```
        path: |
          /var/tmp/test_results/*
          !/var/tmp/test_results/current
```